Dockerfile

```diff
@@ -1,23 +1,25 @@
-FROM node:14-alpine
-
-#Setup folders
-RUN mkdir -p /home/node/app/node_modules && chown -R node:node /home/node/app
-WORKDIR /home/node/app
+# ---- Build ----
+FROM node:14-alpine AS build
 
-#Install node packages
+WORKDIR /usr/src/app
 COPY package*.json ./
-RUN npm install
-
-#Copies source
-COPY . .
-COPY --chown=node:node . .
 
-USER node
+# --ignore-scripts to prevent prisma generate from running.
+# No use until we got the source code
+RUN npm install --ignore-script
 
-EXPOSE 8080
```

Comment:
Guessing this is an old value? On GCP we don't know/care, but locally it's 5050 by default AFAICT

Reply:
I think Cloud Run is actually configured to use 8080, so this change would break it. However, we could consider using a port argument on the Dockerfile.

Reply:
Aha :thinking: [The expose instruction is really only documentation though](https://docs.docker.com/engine/reference/builder/#expose) (the correct ports are exposed regardless), but perhaps GCP uses it for validation or something. Worth checking out none the less :+1:

Reply:
It still starts up on Cloud Run so it seems GCP doesn't really care what the image claims to EXPOSE. We could just skip the EXPOSE instruction altogether but I think it's kinda nice to have locally when doing a `docker ps`. WDYT?

Reply:
Alright, then I think exposing 5050 by default is reasonable!